fairing/preprocessors/base.py

```diff
@@ -103,6 +103,11 @@ def fairing_runtime_files(self):
             dst = os.path.normpath(os.path.join(self.path_prefix, "fairing", f))
             ret[src] = dst
         return ret
+    
+    def is_requirements_txt_file_present(self):
```

Comment:
use a list comprehension here and a Path object instead of just concatenating

Reply:
done. also created #160